cmd/frontend/graphqlbackend/externallink/repository.go

```diff
@@ -44,9 +43,8 @@ func FileOrDir(ctx context.Context, db database.DB, repo *types.Repo, rev, path
 	phabRepo, link, serviceType := linksForRepository(ctx, db, repo)
 	if phabRepo != nil {
 		// We need a branch name to construct the Phabricator URL.
-		branchName, _, _, err := git.ExecSafe(ctx, repo.Name, []string{"symbolic-ref", "--short", "HEAD"})
-		branchName = bytes.TrimSpace(branchName)
-		if err == nil && string(branchName) != "" {
+		branchName, _, err := git.GetDefaultBranchShort(ctx, repo.Name)
+		if err == nil && branchName != "" {
 			links = append(links, NewResolver(
 				fmt.Sprintf("%s/source/%s/browse/%s/%s;%s", strings.TrimSuffix(phabRepo.URL, "/"), phabRepo.Callsign, url.PathEscape(string(branchName)), path, rev),
```

Comment:
Fix unnecessary conversion caught in lint.

```suggestion
				fmt.Sprintf("%s/source/%s/browse/%s/%s;%s", strings.TrimSuffix(phabRepo.URL, "/"), phabRepo.Callsign, url.PathEscape(branchName), path, rev),
```

Reply:
Thanks, I already pushed that while you were reviewing I think.

Reply:
Ah, I see. It doesn't show up here though. :thinking: